gcc.stl.headers.imp

```diff
@@ -157,15 +168,16 @@
   { include: ["<tr1/unordered_map.h>", private, "<tr1/unordered_map>", public ] },
   { include: ["<tr1/unordered_set.h>", private, "<tr1/unordered_set>", public ] },
   { include: ["<tr2/dynamic_bitset.tcc>", private, "<tr2/dynamic_bitset>", public ] },
-  # ( cd /usr/include/x86_64-linux-gnu/c++/8 && grep -r headername | perl -nle 'm/^([^:]+).*@headername\{([^,]*)\}/ && print qq@  { include: ["<$1>", private, "<$2>", public ] },@' | sort -u )
+  # ( cd /usr/include/x86_64-linux-gnu/c++/10 && grep -r headername | perl -nle 'm/^([^:]+).*@headername\{([^,]*)\}/ && print qq@  { include: ["<$1>", private, "<$2>", public ] },@' | sort -u )
   { include: ["<bits/basic_file.h>", private, "<ios>", public ] },
   { include: ["<bits/c++allocator.h>", private, "<memory>", public ] },
-  { include: ["<bits/c++config.h>", private, "<iosfwd>", public ] },
```

Comment:
This one is a little scary -- I wonder if we should map this to both `<iosfwd>` _and_ `<version>` (I'm guessing `<version>` is new in libstdc++10).